webpack/scenes/ContentViews/Create/CreateContentViewFormComponents.js

```diff
@@ -0,0 +1,70 @@
+import { translate as __ } from 'foremanReact/common/I18n';
+import React from 'react';
+import { Popover, Flex, FlexItem } from '@patternfly/react-core';
+import { HelpIcon } from '@patternfly/react-icons';
+import ContentViewIcon from '../components/ContentViewIcon';
+
+
+export const labelComposite = () => (
+  <Flex>
+    <FlexItem spacer={{ default: 'spacerNone' }}><ContentViewIcon composite /></FlexItem>
+    <FlexItem>Composite Content View</FlexItem>
+  </Flex>
+);
+
+export const labelComponent = () => (
+  <Flex>
+    <FlexItem spacer={{ default: 'spacerNone' }}><ContentViewIcon composite={false} /></FlexItem>
+    <FlexItem>Component Content View</FlexItem>
+  </Flex>
+);
+
+export const labelDependencies = () => (
+  <Flex>
+    <FlexItem spacer={{ default: 'spacerSm' }}>Solve Dependencies </FlexItem>
+    <FlexItem>
+      <Popover
+        bodyContent={
+            __('This will solve RPM and Module Stream dependencies on every publish of this Content View. ' +
+            'Dependency solving significantly increases publish time (publishes can take over three times as long) ' +
+            'and filters will be ignored when adding packages to solve dependencies. ' +
+            'Also, certain scenarios involving errata may still cause dependency errors.')
+          }
+        aria-label="Solve Dependencies"
+        position="right"
+      >
+        <button
+          aria-label="More info for dependenices field"
+          onClick={e => e.preventDefault()}
+          className="pf-c-form__group-label-help"
+        >
+          <HelpIcon noVerticalAlign />
+        </button>
+      </Popover>
+    </FlexItem>
+  </Flex>
+);
+
+export const labelAutoPublish = () => (
+  <Flex>
+    <FlexItem spacer={{ default: 'spacerSm' }}>Auto Publish </FlexItem>
+    <FlexItem>
+      <Popover
+        bodyContent={
+            __('Applicable only for composite views. Auto publish composite view when a new version of a component content view is created.' +
+            'Also note auto publish will only happen when the component is marked latest.')
```

Comment:
this sentence was probably there already, but I don't understand what it means

Reply:
So for a composite content view, publishing any child cv which was included with "Always use latest version" option will re-publish the parent as well.

Reply:
I guess the "is marked latest" was a bit confusing to me.

maybe something like this?
```
Automatically publish a new version of the composite content view whenever one of its component content views is published.
(Also note auto-publish will only happen for component views that use the 'Always use latest version' option.)
```

Reply:
Ya this seems better explained. Updated.

Reply:
Good improvement! How do you feel about these tweaks?

> Automatically publish a new version of the composite content view whenever one of its content views is published.
Auto-publish will only happen for component views that use the 'Always use latest version' option.

Removing the parenthesis and the `Also note` as it seems like important enough information to state directly, and removing the 'component' part from content views, we haven't  really settled on a term and I think the context makes it clear that it's a 'child' content view.

Reply:
@johnpmitsch :+1:  to the above

Reply:
Updated with the new text. :+1: